Lib/test/test_dataclasses.py

```diff
@@ -479,6 +498,18 @@ class C:
         self.assertNotEqual(C(3), C(4, 10))
         self.assertNotEqual(C(3, 10), C(4, 10))
 
+    def test_dataclass_params_repr(self):
```

Comment:
This is testing an internal implementation detail. I'm not sure that this should be directly tested.

Reply:
I have no opinion about keeping the test or not. But if you want to keep it, add `@cpython_only` decorator.

Reply:
I have removed the test. I am not sure of a better way to access it publicly.